<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqStreamingControls</class>
 <widget class="QDockWidget" name="pqStreamingControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Streaming Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>391</width>
         <height>728</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="streaming_controls">
          <property name="title">
           <string>General Streaming Controls</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>12</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>show when</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="show_when">
               <property name="toolTip">
                <string>controls whether you see final results or streaming in progress</string>
               </property>
               <item>
                <property name="text">
                 <string>fully drawn</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>each piece drawn</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>cache</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cache_size">
               <property name="toolTip">
                <string>the number of passes each viewable object stores to speed up redisplay</string>
               </property>
               <property name="currentIndex">
                <number>7</number>
               </property>
               <item>
                <property name="text">
                 <string>infinite</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>no</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>pieces</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>number of passes</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="number_of_passes">
               <property name="toolTip">
                <string>the number of pieces the pipeline is broken up into</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>32</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>stop after</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="last_pass">
               <property name="toolTip">
                <string>allows you to stop before the last piece to get faster response time</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>32</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>passes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="stop">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="toolTip">
              <string>halt streaming immediately</string>
             </property>
             <property name="text">
              <string>STOP</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="prioritization_controls">
          <property name="title">
           <string>Prioritized Streaming Controls</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="pipeline_priority">
             <property name="toolTip">
              <string>prioritize pieces bases on data centric properties</string>
             </property>
             <property name="text">
              <string>pipeline prioritization</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="view_priority">
             <property name="toolTip">
              <string>prioritize pieces based on their position in the view area</string>
             </property>
             <property name="text">
              <string>view prioritization</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="refinement_controls">
          <property name="title">
           <string>MultiResolution Streaming Controls</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>refinement depth</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="refinement_depth">
               <property name="toolTip">
                <string>controls the height of the refinement tree</string>
               </property>
               <property name="currentIndex">
                <number>4</number>
               </property>
               <property name="maxVisibleItems">
                <number>15</number>
               </property>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>11</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>12</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>13</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>17</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>18</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>20</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>levels</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>pixel limit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="cell_pixel_factor">
               <property name="toolTip">
                <string>higher numbers mean refinement stops earlier</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>2.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.666000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>backface test</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="back_face_factor">
               <property name="toolTip">
                <string>higher numbers mean more pieces culled</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>-1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>0.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.050000000000000</double>
               </property>
               <property name="value">
                <double>-0.750000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>stop before</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="max_depth">
               <property name="toolTip">
                <string>stops refinement before full resolution to get faster response time</string>
               </property>
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>levels</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>split at most</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="max_splits">
               <property name="toolTip">
                <string>controls the number of pieces to refine on each render</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>8</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>pieces each pass</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="progression_mode">
             <property name="toolTip">
              <string>control refinement manually or let it run on its own</string>
             </property>
             <item>
              <property name="text">
               <string>manual progression</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>automatic progression</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refine">
             <property name="toolTip">
              <string>increase the resolution</string>
             </property>
             <property name="text">
              <string>refine</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="coarsen">
             <property name="toolTip">
              <string>decrease the resolution</string>
             </property>
             <property name="text">
              <string>coarsen</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QCheckBox" name="lock_refinement">
               <property name="text">
                <string>lock</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>refinement of the active object</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="restart_refinement">
               <property name="toolTip">
                <string>completely coarsen the active object</string>
               </property>
               <property name="text">
                <string>restart</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>refinement of the active object</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
